kpops/component_handlers/__init__.py

```diff
@@ -20,3 +34,10 @@ def __init__(
         self.schema_handler = schema_handler
         self.connector_handler = connector_handler
         self.topic_handler = topic_handler
+
+
+def get_handlers() -> ComponentHandlers:
```

Comment:
Cannot be a propety?

Reply:
property of ComponentHandlers? 🤔

Reply:
Yes like this
```suggestion
@property
def component_handlers() -> ComponentHandlers:
```

Reply:
but it would need to be a classmethod property to be globally accessible which is deprecated in Python 3.11+. We have only have a workaround `cached_classproperty` which doesn't type check properly and I'd rather not use it more until there is a better approach.